Editor/FleetParameters.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Amazon.GameLift;
+
+namespace AmazonGameLift.Editor
+{
+    public class FleetParameters
```

Comment:
Is this just for ManagedEC2Fleets? I'd rename the file to `ManagedEC2FleetParameters` to make that clear given we're also introduce Anywhere fleet related functionality with this release.

Reply:
Leaving as is then

Reply:
Actually, chosen to use verbose naming so will rename